usaspending_api/download/lookups.py

```diff
@@ -35,10 +35,6 @@
 from usaspending_api.awards.v2.filters.sub_award import subaward_download
 from usaspending_api.awards.v2.lookups.lookups import award_type_mapping
 
-# Temporary hotfix to get custom award downloads working until a permanent solution can be found
-# from usaspending_api.download.filestreaming.generate_export_query import (
-#     generate_file_b_custom_account_download_export_query,
-# )
```

Comment:
The original PR commented these lines out. Are you sure we want to delete these now?

Reply:
The entire purpose of this pull request is to remove all this stuff.  The ticket is being kicked to the backlog.